frontend/src/features/rollout-announcement/Announcements.tsx

```diff
@@ -1,17 +1,27 @@
-import FirstAnnouncement from './assets/1-dnd.json'
-import SecondAnnouncement from './assets/2-listview.json'
+import FirstAnnouncement from './assets/1-payments.json'
+import SecondAnnouncement from './assets/2-dnd.json'
+import ThirdAnnouncement from './assets/3-listview.json'
 
+// When updating this, remember to update the ROLLOUT_ANNOUNCEMENT_KEY_PREFIX with the new date
+// so admins will see new announcements.
 export const NEW_FEATURES = [
+  {
+    // Announcement date: 2023-05-31
+    title: 'Collect payments on your form',
+    description:
+      'Citizens can now pay for fees and services directly on your form! We integrate with Stripe to provide reliable payments and hassle-free reconciliations. Payment methods we support include credit card and PayNow.',
+    animationData: FirstAnnouncement,
+  },
   {
     title: 'Drag-and-drop form creation',
     description:
       'Add and rearrange fields with ease, and see a preview of your form while editing it.',
-    animationData: FirstAnnouncement,
+    animationData: SecondAnnouncement,
```

Comment:
/noChangeNeeded: we need a better way to do this than naming the assets with positional indicators 😑 . The order is defined in this file (in the array); the assets themselves should only be named on what they contain or are (e.g. `import dndAnnouncement from /assets/new-feature-dnd.json`)

Reply:
Done in [fb6edda](https://github.com/opengovsg/FormSG/pull/6407/commits/fb6eddae42738879705bcf910b7aa9d86d1ec369) along with removing one announcements!